qemu/tests/cfg/virtio_console.cfg

```diff
@@ -229,12 +229,12 @@
                             virtio_port_type_vs1 = serialport
                             virtio_port_params_vs1 = "nr=1"
                         - boot_too_much_ports:
-                            # arm doesn't use pci bus
-                            no aarch64
+                            # arm-mmio doesn't use pci bus
+                            no arm64-mmio
```

Comment:
According to the change in the script file, you want to support the test for `arm64-mmio` machines, so the above line should be removed.

Reply:
I think there are two kinds of types for arm64, 'arm64-mmio' and 'arm64-pci', do they both use pci bus?

Reply:
Nop, I want to support it on s390x. I haven't tested it on arm (although it might work, I don't really care at this point as mmio is not supported)

Reply:
Alright.